src/mlagility/api/script_api.py

```diff
@@ -104,6 +106,8 @@ def benchmark_script(
 
     if devices is None:
         devices = ["x86"]
+    if runtimes is None:
+        runtimes = [SUPPORTED_DEVICES["x86"][0]]
```

Comment:
Suggest creating a named constant, `DEFAULT_RUNTIME=0`, so that this reads: `runtimes = [SUPPORTED_DEVICES["x86"][DEFUALT_RUNTIME]]` (eliminate magic numbers)

Reply:
Suggestion incorporated!